templates/engage/base.html

```diff
@@ -124,31 +123,39 @@ <h1>{{ document["title"] }}</h1>
 
 </section>
 
-{% if document["metadata"]["webinar_date"] is defined %}
-  {% if not document["metadata"]["webinar_date"] | date_has_passed %}
+{% if metadata["webinar_date"] is defined %}
+  {% if not metadata["webinar_date"] | date_has_passed %}
     {% include "engage/shared/_webinar-embed.html" %}
   {% endif %}
 {% else %}
   {% include "engage/shared/_webinar-embed.html" %}
-{% endif %}
 
+<!-- Brighttalk webinar -->
+<section class="p-strip is-shallow" id="register-section">
+  <div class="row" id="webinar">
+    <div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;">
+      <script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : {{channel_id}}, "language": "en-US", "commId" : {{ metadata["webinar_code"] }}, "displayMode" : "standalone", "height" : "auto" }</script>
+      <script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script>
+    </div>
+  </div>
+</section>
+{% endif %}
```

Comment:
I think there's no need for this now, I created a partial for the brighttalk embed (see line 131)

Reply:
Yeah sorry, bad rebase